test/visual/stories.js

```diff
@@ -170,8 +155,11 @@ module.exports = [
     'dialog--alert-error-with-long-title',
     'dialog--fullscreen',
     'dialog--fullscreen-takeover',
-    'dropdown--default',
-    'dropdown--initial-value',
+    'divider--large',
+    'divider--medium',
+    'divider--small',
+    'divider--vertical-small',
+    'divider--vertical-medium',
```

Comment:
Not directly related to your change, but we should add a new story to cover `'divider--vertical-large',` as part of this.